inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -55,6 +56,23 @@ std::string translate_type_name(const std::string& name) {
     return name;
 }
 
+// Some of the operators were added to wrong opsets. This is a
+// mapping that allows such operators to be serialized with proper
+// opsets. If new operators are discovered that have the same problem
+// the opset mapping needs to be updated here. The keys also contain op
+// versions, concatenated after the actual names.
+const std::unordered_map<std::string, std::string>
```

Comment:
Why cannot we use `ngraph::Node::type_info_t` as a key of map?

Reply:
I wasn't sure if it was OK to use it, as it has also the parent type info field besides name and version

Reply:
It will be better to use `NodeTypeInfo` from my point of view.
We already use it as a map key [here](https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/src/pass/graph_rewrite.cpp#L86).
I think we shouldn't have any issues with constructing of such map.

Reply:
I should have looked at how NodeTypeInfo is being compared (now I know that the parent ptr field is not taken into consideration), I applied the changes.